change/@ni-nimble-components-56124785-d76b-4bd0-b330-b457eb4ff9a0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'll make a separate comment, but to accelerate this workflow maybe we should create a new issue template specifically for icons that covers some points:
- Links to the naming scheme documentation
- Adds fields so that the referenced font awesome name can be linked and metaphors pulled from or the alternate name if a font awesome one is not available
- Includes additional metaphors that are needed
- Does not move out of triage / move to ready until those are established and can be merged mechanically

Reply:
Filed tech debt #2263